cmd/webui/ach/ach_js.go

```diff
@@ -78,8 +87,31 @@ func printACH() js.Func {
 	})
 }
 
+func printReadable() js.Func {
+	return js.FuncOf(func(_ js.Value, args []js.Value) interface{} {
+		if len(args) != 1 {
+			return "Invalid no of arguments passed"
```

Comment:
```suggestion
			return "Invalid number of arguments passed"
```
Could we update this in printACH as well?

Reply:
I copied that from the other handlers - I'll change it in all places. I agree "number" is better.